File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self,n):
        aList = []
        i = 1
        while i<=n:
            if i%3==0 and i%5==0:
                aList.append("FizzBuzz")
            elif i%5==0:
                aList.append("Buzz")
            elif i%3==0:
                aList.append("Fizz")
            else:
                aList.append(str(i))
            i+=1
        return aList
            

File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let a = [];
    let i = 1;
    
    while (i <= n) {
        if (i % 3 == 0 && i % 5 == 0) {
            a.push("FizzBuzz");
        } else if (i % 5 == 0) {
            a.push("Buzz");
        } else if (i % 3 == 0) {
            a.push("Fizz");
        } else {
            a.push(i.toString());
        }
        i++;
    }
    
    return a;
};